README.md

```diff
@@ -1,9 +1,18 @@
 # cvs-svc-test-results
 
 #### Run AWS Lambda node functions locally with a mock API Gateway and DynamoDB to test against
+
+Before running the start script, please make sure you have changed your dynamoDB to point locally.
+
+Please refer to [Local Running](#Local-Running) section and also make sure you have dynamoDB credentials set up.
+
+Please request the relevant credentials to be added locally to the `~/.aws/credentials` file.
+
+- `nvm use`
```

Comment:
`nvm use` depends on whether nvm is installed.

Reply:
true but i would expect any developer working with nodeJs to have that installed and see that as I given.
I can add a link to it if you think it is required in case it is missed out.
Let me know, happy to update.